dvc/api/experiments.py

```diff
@@ -31,3 +32,40 @@ def make_checkpoint():
         os.fsync(fobj.fileno())
     while os.path.exists(signal_file):
         sleep(0.1)
+
+
+def exp_save(
+    name: Optional[str] = None,
+    force: bool = False,
+    include_untracked: Optional[List[str]] = None,
+):
+    """
+    Create a new DVC experiment using `exp save`.
+
+    See https://dvc.org/doc/command-reference/exp/save.
+
+    Args:
+        name (str, optional): specify a name for this experiment.
+            If `None`, a default one will be generated, such as `urban-sign`.
+            Defaults to `None`.
+        force (bool):  overwrite the experiment if an experiment with the same
+            name already exists.
+            Defaults to `False`.
+        include_untracked (List[str], optional): specify untracked file(s) to
+            be included in the saved experiment.
+            Defaults to `None`.
+
+    Returns:
+        str: The `Git revision`_ of the created experiment.
+
+    Raises:
+        ExperimentExistsError: If an experiment with `name` already exists and
+            `force=False`.
```

Comment:
I am a bit hesitant with exposing this, I think it's okay to say that it'll fail in this case.

Reply:
It's an internal exception `dvc.repo.experiments.exceptions.ExperimentExistsError`, which we ourselves depend on, and may change (if you think strictly in terms of semver, we cannot break exceptions until a major release).

Reply:
Everything except of dvc.api is internal (although I am not a fan of that, that was the whole point of dvc.api namespace).